plugins/lookup/github_app_access_token.py

```diff
@@ -47,6 +47,13 @@
       - How long the token should last for in seconds.
     default: 600
     type: int
+  github_url:
+    description:
+      - Base URL for the GitHub API (for GitHub Enterprise Server).
+      - "Example: C(https://github-enterprise-server.example.com/api/v3)"
+    default: https://api.github.com
```

Comment:
I saw Felix's comments and this default makes perfect sense, IMHO. What I am missing, though, is the part in the _code_ that will test whether `github_url` has been passed or not, and assign the default value if the latter.
Maybe in line 222?